### PR TITLE
Add user guidance page to support interface regarding GDPR

### PR DESCRIPTION
**Pull request by davidgisbey**
## Context

We need this to be GDPR compliant

## Changes proposed in this pull request

- Add a user guidance page to the support interface

image

## Guidance to review

Content can be found here 

https://docs.google.com/document/d/1dlm54MyV-juFOjw3YAW_Cawbya5XlYExRd_03WJADqc/edit#

## Link to Trello card

https://trello.com/c/LlPe6FWD/1931-add-guidance-to-the-support-console-and-onboarding-packs

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/support_interface/guidance/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<% content_for :title, t('page_titles.guidance') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <p class='govuk-body'>Within the support console there is sensitive information and personal data about the users of the Apply for teacher training service which we store inline with our Data Protection Agreement and Privacy Notice.</p>
```
Comment (paulrobertlloyd) [inline app/views/support_interface/guidance/index.html.erb:6]: Can we be consistent and use double quotes around HTML attributes?

File: app/views/support_interface/guidance/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<% content_for :title, t('page_titles.guidance') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
```
Comment (paulrobertlloyd) [inline app/views/support_interface/guidance/index.html.erb:5]: Remove empty new lines

File: app/views/support_interface/guidance/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    <p class='govuk-body'>Within the support console there is sensitive information and personal data about the users of the Apply for teacher training service which we store inline with our Data Protection Agreement and Privacy Notice.</p>
+
+    <p class='govuk-body'>As a user of the Apply support console there are specific things you should be aware of and practices to ensure you use data in the appropriate manner and we are not at risk of mis-use or breach of GDPR and other data security policies.</p>
+
+    <p class='govuk-body'>Most practice in line with GDPR is common sense but the key messages are:
+      <ul class='govuk-list govuk-list--bullet'>
+        <li>only access what data you need to</li>
+        <li>think whether there is a reason to view what you’re accessing</li>
+        <li>don't copy data and store in other locations</li>
```
Comment (paulrobertlloyd) [inline app/views/support_interface/guidance/index.html.erb:14]:
```suggestion
        <li>don’t copy data and store in other locations</li>
```

Replace dumb quotes with smart ones.

File: app/views/support_interface/guidance/index.html.erb (hunk 2 above)
Comment (davidgisbey) [reply app/views/support_interface/guidance/index.html.erb:5]: have actioned all of these 👍

File: app/views/support_interface/guidance/index.html.erb (hunk 3 above)
Comment (paulrobertlloyd) [inline app/views/support_interface/guidance/index.html.erb:14]: Actually given #2698, in this particular instance, use `do not`, not `don’t`.

Comment (davidgisbey) [reply app/views/support_interface/guidance/index.html.erb]: Yea good point 👍 i def shouldn't be blindly copy and pasting from the doc. My bad.

File: app/views/support_interface/guidance/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<% content_for :title, t('page_titles.guidance') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <p class="govuk-body">Within the support console there is sensitive information and personal data about the users of the Apply for teacher training service which we store inline with our Data Protection Agreement and Privacy Notice.</p>
+
+    <p class="govuk-body">As a user of the Apply support console there are specific things you should be aware of and practices to ensure you use data in the appropriate manner and we are not at risk of mis-use or breach of GDPR and other data security policies.</p>
+
+    <p class="govuk-body">Most practice in line with GDPR is common sense but the key messages are:
+      <ul class='govuk-list govuk-list--bullet govuk-!-margin-left-2  govuk-!-margin-right-2'>
```
Comment (paulrobertlloyd) [inline app/views/support_interface/guidance/index.html.erb:10]:
```suggestion
      <ul class="govuk-list govuk-list--bullet govuk-!-margin-left-2 govuk-!-margin-right-2">
```

File: config/locales/en.yml [hunk 5]
```diff
@@ -140,6 +140,7 @@ en:
     remove_course_choice: Remove your course choice
     withdraw_application: Are you sure you want to withdraw your application?
     gcse_naric_reference: Do you have a NARIC statement of comparability for your maths qualification?
+    guidance: User Guidance
```
Comment (paulrobertlloyd) [inline config/locales/en.yml:143]:
```suggestion
    guidance: User guidance
```

**Pull request merged**